### ISSUE TITLE
CSS WG rechartering

### ISSUE THREAD

**Issue by svgeesus** [w3c/strategy#338]
# Charter Review

[Draft charter](https://w3c.github.io/charter-drafts/2022/css-2022.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* Existing 
 - [x] Existing WG recharter 

[diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2020%2F12%2Fcss-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2022%2Fcss-2022.html)

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)
- [CSS WG GitHub issue](https://github.com/w3c/csswg-drafts/issues/7468)

Anything else we should think about as we review? 
- See discussion at https://github.com/w3c/csswg-drafts/issues/7468

**Comment by himorin** [comment]
history table lacks a link to the current running charter as https://www.w3.org/2020/12/css-wg-charter.html (at bottom line)?

**Comment by ruoxiran** [comment]
We are looking forward to discussing accessible navigation on TPAC, there is no other comment from APA WG.

**Comment by svgeesus** [comment]
> lacks a link to the current running charter

Thanks for noticing, will fix.

**Comment by svgeesus** [comment]
@himorin any other comments from an I18n perspective? Are you the one reviewing the charter, or did you just happen to notice the missing link?

**Comment by svgeesus** [comment]
@samuelweiler any comments from your perspective?

**Comment by svgeesus** [comment]
@wseltzer I discussed W3M review with @PLH and he agreed to be one reviewer; we also thought that W3M review could start even before horizontal review is complete.

Could you be the other reviewer? (I know you are very busy now with LE bylaws and such, so "no" is a perfectly acceptable answer)

**Comment by himorin** [comment]
no comment/request from i18n. (Sorry for delay on responding after 1 week internal discussion slot)

**Comment by svgeesus** [comment]
> Sorry for delay on responding

No problem at all. I'm hoping to have this charter discussed at the 24 Aug W3M, so thanks for completing your review well before that!

**Comment by svgeesus** [comment]
@swickr is the second reviewer.

**Comment by svgeesus** [comment]
@samuelweiler any comments? If not, please mark the privacy and security review as complete?

**Comment by samuelweiler** [comment]
With apologies for the delay, no comments.

**Comment by svgeesus** [comment]
@plh I believe both your comments are addressed.

@swickr did you have comments? (If sent by email my apologies, I still have a backlog)

**Comment by plehegar** [comment]
(waiting to start the AC review)

**Comment by svgeesus** [comment]
[Formal objection from fantasai](https://lists.w3.org/Archives/Member/w3c-ac-forum/2022OctDec/0027.html)

Suggested replacement text looks fine, but this is a comment on the charter template not specifically on the CSS WG proposed charter.

**Comment by svgeesus** [comment]
Formal objection resolved, w3m approval requested.

**Comment by svgeesus** [comment]
w3m approved 11 Jan 2023
